src/rebuild-module-cache.coffee

```diff
@@ -20,7 +20,7 @@ class RebuildModuleCache extends Command
       Usage: apm rebuild-module-cache
 
       Rebuild the module cache for all the packages installed to
-      ~/.atom/packages
+      ~/.range/packages
```

Comment:
```suggestion
      ~/.pulsar/packages
```

This seems like the value we would expect here?

Reply:
Great catch